homeassistant/components/zwave_js/__init__.py

```diff
@@ -331,8 +331,8 @@ async def async_setup_platform(self, platform: Platform) -> None:
         """Set up platform if needed."""
         if platform not in self.platform_setup_tasks:
             self.platform_setup_tasks[platform] = self.hass.async_create_task(
-                self.hass.config_entries.async_forward_entry_setups(
-                    self.config_entry, [platform]
+                self.hass.config_entries.async_forward_entry_setup(
```

Comment:
We don't want to change this.

Reply:
OK, I kept getting a failure without this but will revert.

Reply:
works now 🤷🏾